ExpoImplementationEAS/ios/Podfile.lock

```diff
@@ -629,9 +629,9 @@ SPEC CHECKSUMS:
   React-utils: 7a9918a1ffdd39aba67835d42386f592ea3f8e76
   ReactCommon: a77b3916d0d39113b5f0de714a0acd47c9d17c3a
   SocketRocket: f32cd54efbe0f095c4d7594881e52619cfe80b17
-  SwiftProtobuf: bcfd2bc231cf9ae552cdc7c4e877bd3b41fe57b1
+  SwiftProtobuf: 69f02cd54fb03201c5e6bf8b76f687c5ef7541a3
   Yoga: 86fed2e4d425ee4c6eab3813ba1791101ee153c6
 
 PODFILE CHECKSUM: 79bb40f439bd18e788decd9c1a77ac2bfd937f2a
 
-COCOAPODS: 1.11.3
+COCOAPODS: 1.14.2
```

Comment:
@cedancp-cs  I would suggest also updating Gemfile and Gemfile.lock to this version, so we are consistent with cocoapods versions across machines